README.md

```diff
@@ -1,260 +1,20 @@
-- **Create interview**
-  - **POST** `/interviews`
-  - Creates a new interview.
-  - **Responses:**
-    - `201 Created` - Interview created successfully
-    - `400 Bad Request` - Invalid interview data
-- **Save question**
-  - **POST** `/interviews/question`
-  - Saves a new question for an interview.
-  - **Responses:**
-    - `200 OK` - Question saved successfully
-    - `400 Bad Request` - Invalid question data
-- **Evaluate question**
-  - **PATCH** `/interviews/question/grade`
-  - Evaluates (grades) a specific interview question.
-  - **Responses:**
-    - `200 OK` - Question evaluated successfully
-    - `400 Bad Request` - Invalid question data
-- **Get questions by skill ID**
-  - **GET** `/interviews/questions/skill/{id}`
-  - Retrieves questions related to a specific skill ID.
-  - **Responses:**
-    - `200 OK` - Questions found
-- **Update feedback**
-  - **PATCH** `/interviews/{interviewId}/feedback`
-  - Updates feedback for a specific interview.
-  - **Responses:**
-    - `200 OK` - Feedback updated successfully
-    - `400 Bad Request` - Invalid input
-
-### Skill Management Endpoints
-- **Get skill tree**
-  - **GET** `/skills`
-  - Retrieves the skill tree.
-  - **Responses:**
-    - `200 OK` - Skill tree found successfully
-    - `401 Unauthorized` - Unauthorized access
-- **Add user skills**
-  - **POST** `/user/skills`
-  - Adds skills for a user based on role ID.
-  - **Responses:**
-    - `200 OK` - User skills added successfully
-    - `401 Unauthorized` - Unauthorized access
-
-### User Management Endpoints
-- **Get information about all users**
-  - **GET** `/users`
-  - Retrieves information about all users with pagination support.
-  - **Responses:**
-    - `200 OK` - Users retrieved successfully
-- **Get information about a user by email**
-  - **POST** `/users/by-email`
-  - Retrieves information about a user based on their email address.
-  - **Responses:**
-    - `200 OK` - User found
-    - `404 Not Found` - User not found
-- **Get information about a user by ID**
-  - **GET** `/users/{id}`
-  - Retrieves information about a user based on their ID.
-  - **Responses:**
-    - `200 OK` - User found
-    - `404 Not Found` - User not found
-- **Change user password**
-  - **PATCH** `/users/password`
-  - Changes the password of the authenticated user.
-  - **Responses:**
-    - `200 OK` - Password changed successfully
-- **Delete the authenticated user**
-  - **DELETE** `/users`
-  - Deletes the authenticated user's account.
-  - **Responses:**
-    - `200 OK` - User deleted successfully
-
-## Authorization Rules
-### User
-**Permissions:**
-  - Register and update personal profile with skills.
-  - Participate in mock interviews.
-  - View their own interview results and feedback.
-  - Receive interview reminders and notifications.
-### Admin
-**Permissions:**
-  - Manage user accounts (create, edit, delete users).
-  - View and manage all interviews and questions.
+- [Functionality](/docs/functionality.md)
```

Comment:
The link doesn't work in IntelliJ, remove the first slash

Reply:
Removed first slash. Fixed